Lesson_23/task1.py

```diff
@@ -0,0 +1,80 @@
+"""
+Task 1 - Big O
+We assume that all lists passed to functions are the same length N
+Match big O complexities with the code snippets below
+
+Answers:
+1 - n       - question1
+2 - 1       - question2
+3 - n^2     - question3
+4 - n       - question4
+5 - n^2     - question5
+6 - log n   - question6
+"""
+
+from typing import List, Tuple
+
+
+def question1(first_list: List[int], second_list: List[int]) -> List[int]:
+    """O(1+n(a+1))
```

Comment:
That's correct, but just a small addition from me: if we have multiple parameters in our function, we normally count BigO for each of them (like O(n*m) where n - size of the first list, m - size of the second list). This answer wasn't present in the options, so your answer is totally correct